Apps/Contoso.UWP.Puppet/MainPage.xaml.cs

```diff
@@ -7,6 +7,7 @@
 using System.Threading.Tasks;
 using Windows.UI.Xaml;
 using Windows.UI.Xaml.Controls;
+using UnhandledExceptionEventArgs = Windows.UI.Xaml.UnhandledExceptionEventArgs;
```

Comment:
What is this change?

Reply:
https://github.com/microsoft/appcenter-sdk-dotnet/pull/1278#discussion_r363648179